### PR TITLE
[singlejar] Remove unused port.cc and actually test desugar_checking_test

### PR DESCRIPTION
**Pull request by rongjiecomputer**
#2241

### CODE REVIEW

**Comment by laszlocsomor** [comment]
Sorry for the long delay in review. We're in NYC this week for BazelCon.

File: src/tools/singlejar/BUILD [hunk 1]
```diff
@@ -100,7 +99,7 @@ cc_test(
 cc_test(
     name = "desugar_checking_test",
     srcs = [
-        "combiners_test.cc",
+        "desugar_checking_test.cc",
```
Comment (laszlocsomor) [inline src/tools/singlejar/BUILD]: Hm, was `desugar_checking_test` never actually executing?

Comment (rongjiecomputer) [reply src/tools/singlejar/BUILD]: I think it never actually executed. See the commit that introduced `desugar_checking` library: https://github.com/bazelbuild/bazel/commit/5f911199c250e77a832be2e4f69017c69d446273

**Pull request closed without merging**